db/migrate/20190402154937_add_bill_id_to_items.rb

```diff
@@ -0,0 +1,5 @@
+class AddBillIdToItems < ActiveRecord::Migration[5.2]
+  def change
+    add_column :items, :bill_id, :integer	  
```

Comment:
Please change this to create a foreign key: https://guides.rubyonrails.org/active_record_migrations.html#foreign-keys.

Reply:
First I tried to make it a foreign key in 20190328193948_change_items.rb in an up method.  That didn't work so I tried it in 20190401150038_change_items2.rb in a change method.  Neither worked so I added it as a regular column here 20190402154937_add_bill_id_to_items.rb.  Finally i tried one more time as a foreign key here 20190402182736_change_items3.rb and it's still not working.  Any ideas why?